client/question/question.js

```diff
@@ -514,17 +514,19 @@ tie.constant('WRONG_LANGUAGE_ERRORS', {
  * @type {{}}
  */
 tie.constant('SYSTEM_GENERATED_TIPS', {
-  python: [{
-    // Included a print statement
-    type: 'print',
-    regexString: '\\bprint\\b',
-    message: [
-      'We noticed that you\'re using a print statement within your code. ',
-      'Since you will not be able to use such statements in a technical ',
-      'interview, TIE does not support this feature. We encourage you to ',
-      'instead step through your code by hand.'
-    ].join('')
-  }]
+  python: []
+  // The system generated tip when stdout is disabled.
```

Comment:
Either remove the code entirely or keep it in. Don't submit commented-out code.

If we're controlling the display by a flag then perhaps we will need an additional option for each tip type, along the lines of "requirePrintToBeDisabled" or similar.

Reply:
Actually I would suggest (if it's OK) to have a single field still. The problem with a dictionary is that, as you're seeing, it's already getting a bit complicated. So I would introduce this complexity only when it's clear we need it.

Also, can we not have the key be optional? It's better to have each dict have a fixed set of required keys. It makes it possible for the code to act on a standardly-formatted dict without having to check presence of the key.